<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>656</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>source/window_24_24.png</normaloff>source/window_24_24.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>88</y>
      <width>169</width>
      <height>537</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_left"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>176</x>
      <y>95</y>
      <width>849</width>
      <height>417</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_middle"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>176</x>
      <y>512</y>
      <width>849</width>
      <height>113</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_buttom"/>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>632</y>
      <width>553</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>176</x>
      <y>16</y>
      <width>833</width>
      <height>65</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_Go_Backstage">
     <property name="geometry">
      <rect>
       <x>696</x>
       <y>24</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>查看详细信息</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_1">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>8</y>
      <width>124</width>
      <height>73</height>
     </rect>
    </property>
    <property name="title">
     <string>选择模块</string>
    </property>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>16</y>
       <width>94</width>
       <height>40</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_1">
      <item>
       <widget class="QComboBox" name="comboBox_Select_Module">
        <item>
         <property name="text">
          <string>LX 8PT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LX 8TC</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
